docs: replace LXD for Multipass on the tutorials (#2344)
canonical/ubuntu-advantage-client: docs/tutorials/fix_scenarios.md

## Prerequisites

> On this tutorial, you will use [Multipass](https://multipass.run/) virtual machines (VM).

Reviewer: ```suggestion
In this tutorial, you will use [Multipass](https://multipass.run/) virtual machines (VM).
```

CVE/USN is fully fixed in the machine.

> You can better see this on an `pro fix` call that does fix a package. Let's install a package

Reviewer: ```suggestion
You can better see this in a `pro fix` call that does fix a package. Let's install a package
```
